core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -629,10 +629,8 @@ private StorageDirView allocateSpace(long sessionId, AllocateOptions options)
           // Block expansion are forcing the location. We do not want the review's opinion.
           dirView = mAllocator.allocateBlockWithView(sessionId, options.getSize(),
               options.getLocation(), allocatorView.refreshView(), true);
-          if (LOG.isDebugEnabled()) {
-            LOG.debug("Allocation after freeing space for block expansion. availableBytes: {}",
-                dirView.getAvailableBytes());
-          }
+          LOG.debug("Allocation after freeing space for block expansion. availableBytes: {}",
```

Comment:
I don't think you should print 0 when this is null. As far as i remember the `dirView` being null means there's no available allocation or the allocation is rejected. The dir is not necessarily 0 byte availability. Therefore printing 0 here may be confusing.

I will suggest we do a null check and just `LOG.debug` a message saying there's no available allocation found if it's null. When it's not null we just print the same message.

Reply:
Thanks for review!
How do you feel the way now? I found there are lots of  `if` conditions here. so i try not to introduce introduce another `if` statement.